builder/parser/line_parsers.go

```diff
@@ -235,7 +246,9 @@ func parseJSON(rest string) (*Node, map[string]bool, error) {
 // so, passes to parseJSON; if not, quotes the result and returns a single
 // node.
 func parseMaybeJSON(rest string) (*Node, map[string]bool, error) {
-	rest = strings.TrimSpace(rest)
```

Comment:
@tiborvass because I realized that "rest" (which are the args of the cmd) went thru splitCommand() which already trimmed the spaces.